lxd/daemon.go

```diff
@@ -447,7 +451,29 @@ func (d *Daemon) handleOIDCAuthenticationResult(result *oidc.AuthenticationResul
 			return fmt.Errorf("Failed to update OIDC identity information: %w", err)
 		}
 
-		updateIdentityCache(d)
+		action = lifecycle.IdentityUpdated
+	}
+
+	if action != "" {
+		// Notify other nodes about the new identity.
+		s := d.State()
+		notifier, err := cluster.NewNotifier(s, s.Endpoints.NetworkCert(), s.ServerCert(), cluster.NotifyAlive)
+		if err != nil {
+			return fmt.Errorf("Failed to notify cluster members of new or updated OIDC identity: %w", err)
+		}
+
+		err = notifier(func(client lxd.InstanceServer) error {
```

Comment:
Does this mean that if one of the cluster members if down nobody can login via oidc?

Reply:
As we discussed, we're using `cluster.NotifyAlive` to only notify cluster members are currently deemed to be online. If that fails the user will need to try again later. When other members come back online, they will update their cache via the heartbeat handler which performs the update if the members state has changed.